envoy/auth_server/main.go

```diff
@@ -25,15 +25,19 @@ import (
 // TODO_CONSIDER(@commoddity): Make this configurable. See thread here: https://github.com/buildwithgrove/path/pull/52/files/1a3e7a11f159f5b8d3c414f2417f7879bcfab410..258136504608c1269a27047bb9bded1ab4fefcc8#r1859409934
 const port = 10003
 
+// TODO_TECHDEBT(@commoddity): Make these values part of PATH's config YAML and remove the dependency on environment variables.
```

Comment:
Any chance we can make this an MVP or an upnext?

This doesn't seem like THAT much work.

Reply:
Done.